### DISCUSSION TITLE
Which version allows for /inference to respond?

### DISCUSSION THREAD

**Discussion by POD-io** [openai/whisper#2145] [Q&A]
I'm working with another program that expects to be able to send voice data to 127.0.0.1:8080/inference -- but this is not hosted on the versions I've installed. This one, including whispercpp give a "file not found" on that page, but a 'hello' on the main page

Anyone familiar with which version hosts the /inference page?

**Comment by phineas-pta** [comment]
no idea what u doing 🤔

**Reply by POD-io** [reply]
An external program is calling for the /inference page, which I am told is a part of whisper. However I have not found this page in any of the implementations of whisper. Was hoping someone more well versed would know what the /inference page is, or know which whisper codebase might have this page included.

More specifically, it is passing a .wav file to this /inference page in order to get a STT transcription back as a reply.

**Reply by POD-io** [reply]
It looks like it is detailed on the whispercpp page, I'm going to experiment there

https://github.com/ggerganov/whisper.cpp/tree/master/examples/server
